src/app/stacktrace_win.h

```diff
@@ -262,6 +334,8 @@ const QString straceWin::getBacktrace()
     //logStream << "\n\nList of linked Modules:\n";
     //EnumModulesContext modulesContext(hProcess, logStream);
     //SymEnumerateModules64(hProcess, EnumModulesCB, (PVOID)&modulesContext);
+    SymCleanup(hProcess);
```

Comment:
@SeigneurSerpent:  Is this still needed now that the lines above are commented out?